ThirdParty/Bolts/BoltsInClasspath/README.md

```diff
@@ -0,0 +1,77 @@
+﻿The sample uses weather.com free subscription rest API to demonstrate the Bolts usage. Please register for free and use the token generated from https://home.openweathermap.org/users/sign_up and replace it with apiKey placeholder in the Constants.m file
```

Comment:
The readme should have at least a title and use GitHub markdown to make it more readable by having important text in bold and Objective-C code properly highlighted. See https://github.com/Microsoft/WinObjC-Samples/tree/master/Scenarios/Radial%20Control or any other readme from the scenario folder for good examples 